src_python/habitat_sim/simulator.py

```diff
@@ -276,19 +521,30 @@ def __set_from_config(self, config: Configuration) -> None:
 
         self._default_agent_id = config.sim_cfg.default_agent_id
 
-        self.__sensors: List[Dict[str, Sensor]] = [
-            dict() for i in range(len(config.agents))
-        ]
+        self.__noise_models: Dict[str, SensorNoiseModel] = {}
         self.__last_state = dict()
-        for agent_id, agent_cfg in enumerate(config.agents):
-            for spec in agent_cfg.sensor_specifications:
-                self._update_simulator_sensors(spec.uuid, agent_id=agent_id)
+        for agent_id, _ in enumerate(config.agents):
```

Comment:
How will this allow agents to be created with different sensor configs?
Also
```suggestion
        for agent_id in range(len(config.agents)):
```